spec/licensee/projects/github_project_spec.rb

```diff
@@ -0,0 +1,140 @@
+RSpec.describe Licensee::Projects::GitHubProject do
```

Comment:
One idea to make this a tighter integration tests with the other project types would be to move one of the fixtures to a submodule and point the test directly to that repo when exercising this project type (or locally for the other two).